source/manual/cdn.html.md

```diff
@@ -180,49 +143,36 @@ done
 [firewall rules]: https://github.com/alphagov/govuk-provisioning/blob/master/vcloud-edge_gateway/vars/production_carrenza_vars.yaml
 [vcl_config]: https://github.com/alphagov/govuk-cdn-config/
 [check-cdn-ip-ranges]: https://deploy.publishing.service.gov.uk/job/Check_CDN_IP_Ranges/
-[raise-with-re]: raising-issues-with-reliability-engineering.html
+[raise-with-re]: https://docs.publishing.service.gov.uk/manual/raising-issues-with-reliability-engineering.html
 [vcd-cli]: https://github.com/vmware/vcd-cli
 [carrenza-vpn]: https://docs.publishing.service.gov.uk/manual/connect-to-vcloud-director.html#connecting-with-cisco-anyconnect
 
 ## Banning IP addresses at the CDN edge
 
-We occasionally decide to ban an IP address at our CDN edge if they exhibit the
-following behaviour:
+We occasionally decide to ban an IP address at our CDN edge if they exhibit the following behaviour:
 
 - not respecting [our robots.txt directives][robots]
-- repeatedly receiving 429 (rate limit) error responses from origin and not
-  slowing down
+- repeatedly receiving 429 (rate limit) error responses from origin and not slowing down
 - making suspicious requests like attempting SQL injection queries
 
 [robots]: https://www.gov.uk/robots.txt
 
-Banning IPs shouldn't be taken lightly as IP address can be shared my multiple
-user devices and the user behind an IP address can change over time, so there's
-always a chance that we may block a legitimate user when we ban IP addresses.
+Banning IPs shouldn't be taken lightly as IP address can be shared my multiple user devices and the user behind an IP address can change over time, so there's always a chance that we may block a legitimate user when we ban IP addresses.
 
-You can change the list of banned IP addresses by modifying the
-[YAML config file][ip_ban_config] and [deploying the configuration][ip_ban_deploy].
+You can change the list of banned IP addresses by modifying the [YAML config file][ip_ban_config] and [deploying the configuration][ip_ban_deploy].
 
 [ip_ban_config]: https://github.com/alphagov/govuk-cdn-config-secrets/blob/master/fastly/dictionaries/config/ip_address_blacklist.yaml
 [ip_ban_deploy]: https://deploy.publishing.service.gov.uk/job/Update_CDN_Dictionaries/build
 
 ## Bouncer's Fastly service
 
-A Fastly CDN service can normally handle up to 1000 domains (this limit was
-undocumented).
+A Fastly CDN service can normally handle up to 1000 domains (this limit was undocumented).
```

Comment:
I assume it still is? And if it isn't, we should just remove the parentheses:

```suggestion
A Fastly CDN service can normally handle up to 1000 domains (this limit is undocumented).
```

Reply:
Hmm, not sure if it still is or not as I don't have access to the ZenDesk ticket about it. I've made the change as it makes more sense.